config.sub

```diff
@@ -1042,7 +1042,7 @@ case $basic_machine in
 		;;
 	ppc64)	basic_machine=powerpc64-unknown
 		;;
-	ppc64-* | ppc64p7-*) basic_machine=powerpc64-`echo "$basic_machine" | sed 's/^[^-]*-//'`
+	ppc64-*) basic_machine=powerpc64-`echo "$basic_machine" | sed 's/^[^-]*-//'`
```

Comment:
Note that this part of the patch is gone but also not needed in Fedora any more. If this actually is needed in Fedora downstream, it should be moved to Python upstream instead of kept as a patch.